### PR TITLE
[Excel] (Events) (Worksheets) Add onFormulaChanged conceptual docs

### PR DESCRIPTION
**Pull request by alison-mk**
This PR adds: 
- A note about onFormulaChanged in the Events article
- An article section about onFormulaChanged in the Worksheets article

### CODE REVIEW

File: docs/excel/excel-add-ins-worksheets.md [hunk 1]
```diff
@@ -313,6 +313,53 @@ function onWorksheetChanged(eventArgs) {
 }
 ```
 
+## Detect formula changes (preview)
+
+> [!NOTE]
+> The `Worksheet.onFormulaChanged` event is currently only available in public preview. [!INCLUDE [Information about using preview APIs](../includes/using-excel-preview-apis.md)]
+> 
+
+Your add-in can track changes to formulas in a worksheet. This may be useful when a worksheet is connected to an external database and formula changes in the worksheet need to trigger corresponding updates in the external database.
```
Comment (AlexJerabek) [inline docs/excel/excel-add-ins-worksheets.md:322]: A couple suggestions to make this sound more scenario-driven.
```suggestion
Your add-in can track changes to formulas in a worksheet. This is useful when a worksheet is connected to an external database. When the formula changes in the worksheet, the event in this scenario triggers corresponding updates in the external database.
```

File: docs/excel/excel-add-ins-worksheets.md [hunk 2]
```diff
@@ -313,6 +313,53 @@ function onWorksheetChanged(eventArgs) {
...
+
+To detect changes to formulas, [register an event handler](excel-add-ins-events.md#register-an-event-handler) for the [onFormulaChanged](/javascript/api/excel/excel.worksheet#onFormulaChanged) event of a worksheet.  Event handlers for the `onFormulaChanged` event receive a [WorksheetFormulaChangedEventArgs](/javascript/api/excel/excel.worksheetformulachangedeventargs) object when the event fires.
```
Comment (AlexJerabek) [inline docs/excel/excel-add-ins-worksheets.md:324]: Extra space.
```suggestion
To detect changes to formulas, [register an event handler](excel-add-ins-events.md#register-an-event-handler) for the [onFormulaChanged](/javascript/api/excel/excel.worksheet#onFormulaChanged) event of a worksheet. Event handlers for the `onFormulaChanged` event receive a [WorksheetFormulaChangedEventArgs](/javascript/api/excel/excel.worksheetformulachangedeventargs) object when the event fires.
```

**Review by AlexJerabek — APPROVED** [review]
Approving with a couple suggestions.

File: docs/excel/excel-add-ins-worksheets.md (hunk 1 above)
Comment (alison-mk) [reply docs/excel/excel-add-ins-worksheets.md:322]: Thank you!

**Pull request merged**